gen3/bin/kube-setup-secrets.sh

```diff
@@ -440,29 +428,6 @@ if [[ -f "$(gen3_secrets_folder)/creds.json" ]]; then  # update secrets
       PGPASSWORD="$sheepdog_db_password" psql -t -U "$sheepdog_db_user" -h $gdcapi_db_host -d $gdcapi_db_database -c "$sql" || true
     fi
   fi
-
-  # setup the database ...
-  cd "$(gen3_secrets_folder)"
-  if [[ ! -f .rendered_gdcapi_db ]]; then
```

Comment:
why did you remove this block?

Reply:
ooops I don't know what happened. Will add it back.